.circleci/config.yml

```diff
@@ -212,6 +212,25 @@ jobs:
       - run:
           command: ./.ci/validation.sh git-diff
 
+  run-inspections:
+    docker:
+      - image: checkstyle/idea-docker:jdk11-idea2022.2.2
+
+    steps:
+      - checkout
+      - run:
+          name: Run inspections
+          command: |
+            mkdir -p .idea
+            mv config/intellij-idea-inspections-misc.xml \
+              .idea/misc.xml
+            ls .idea
+            ./.ci/idea_inspection.sh
+          no_output_timeout: 25m
```

Comment:
Do we really expect that long execution ?
Can we just double of what we have now?

Reply:
Ok, make sense, I remember you told me that execution is only few minutes by CLI :). Ok it is kind same as in TC, that is good.

Reply:
We can make execution faster by increasing resource class, if you want to. It ran in 4-5 minutes on large (default is medium for us).